agent-lib/src/main/java/org/terracotta/angela/common/cluster/AtomicBoolean.java

```diff
@@ -15,12 +15,16 @@
  */
 package org.terracotta.angela.common.cluster;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import org.apache.ignite.Ignite;
 import org.apache.ignite.IgniteAtomicLong;
 
-public class AtomicBoolean {
+import java.io.Serializable;
```

Comment:
why is made serializable ?

Reply:
Example:
```
      final Barrier started = factory.cluster().barrier("started", 2);
      final ClientArrayFuture future = factory.clientArray(0).executeOnAll(cluster -> {
        [...]
      });
```
Ignite requires serialisable closures, and so any variable shared by the test and a client job has to be serialisable.

Reply:
Note: it was working before. It's just more correct now.